Frameworks/Foundation/NSURLConnection.mm

```diff
@@ -329,7 +325,11 @@ - (void)setDelegateQueue:(NSOperationQueue*)queue {
 */
 - (void)unscheduleFromRunLoop:(NSRunLoop*)aRunLoop forMode:(NSRunLoopMode)mode {
     @synchronized(self) {
-        auto it = std::find(_scheduledRunLoops.begin(), _scheduledRunLoops.end());
+        auto it = std::find_if(_scheduledRunLoops.begin(),
+                               _scheduledRunLoops.end(),
+                               [aRunLoop](std::pair<StrongId<NSRunLoop>, StrongId<NSRunLoopMode>>& pair) {
+                                   return aRunLoop == pair.first.get();
```

Comment:
nit: AutoId should have an operator== for T (but I wouldn't be surprised if it didn't)